apps/ledger-live-desktop/src/renderer/families/bitcoin/AccountHeaderManageActions.ts

```diff
@@ -7,13 +7,18 @@ import { stakeDefaultTrack } from "~/renderer/screens/stake/constants";
 import { BitcoinAccount } from "@ledgerhq/coin-bitcoin/lib/types";
 import { TokenAccount } from "@ledgerhq/types-live";
 import IconCoins from "~/renderer/icons/Coins";
+import useFeature from "@ledgerhq/live-common/featureFlags/useFeature";
 
 type Props = {
   account: BitcoinAccount | TokenAccount;
   parentAccount: BitcoinAccount | undefined | null;
 };
 
 const AccountHeaderActions = ({ account, parentAccount }: Props) => {
+  const stakeProgramsFeatureFlag = useFeature("stakePrograms");
+  const listFlag = stakeProgramsFeatureFlag?.params?.list ?? [];
+  const stakeProgramsEnabled = stakeProgramsFeatureFlag?.enabled ?? false;
```

Comment:
```suggestion
const stakeProgramsEnabled = stakeProgramsFeatureFlag?.enabled
```

Reply:
imho it makes sense to force a boolean here instead of possible undefined. But open to correction.